src/react-components/src/components/02-molecules/Cards/SearchResultItem.tsx

```diff
@@ -0,0 +1,88 @@
+/* MC-25 Search Results Item */
+import * as React from "react";
+import bem from "../../../utils/bem";
+import IconLink from "../../01-atoms/Links/IconLink";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import EditionCard from "./EditionCard";
+import UnderlineLink from "../../01-atoms/Links/UnderlineLink";
+
+export type EditionInfo = {
+  editionYearHeading: JSX.Element,
+  publisherAndLocation: string,
+  coverUrl: string,
+  language: string,
+  license: string,
+  readOnlineLink: string,
+  downloadLink: string
+};
+
+export interface SearchResultItemProps {
+  id: string;
+
+  resultIndex: number;
+
+  modifiers?: [];
+  blockName?: string;
+
+  headingContent: JSX.Element;
+
+  subtitleText: string;
+
+  authorLinkElement: JSX.Element;
+
+  editionInfo: EditionInfo;
+  editionsLinkElement: JSX.Element;
+}
+
+export default class SearchResultItem extends React.Component<SearchResultItemProps, {}> {
+  constructor(props: SearchResultItemProps) {
+    super(props);
+  }
+
+  render(): JSX.Element {
+    const { id,
+      resultIndex,
+      modifiers, blockName, headingContent,
+      subtitleText,
+      authorLinkElement,
+      editionInfo, editionsLinkElement} = this.props;
+    const baseClass = "search-result-item";
+
+    try {
```

Comment:
The one-child test is being done here but `this.props.children` isn't being rendered in the final component so it doesn't seem necessary (and there's no example of it being used in the stories.

Reply:
Okay! I get this now. This is for `headingContent` to check that it is just one element that is being passed but _not_ for `SearchResultItem`. This should be changed to:
```
    try {
      React.Children.only(headingContent.props.children as React.ReactElement);
    } catch (e) {
      const children = React.Children.map(headingContent.props.children, child =>
        (child as JSX.Element).type);
      // Catching the error because React's error isn't as helpful.
      throw new Error(`Please only pass one child into headingContent, got ${children.join(", ")}`);
    }
```

There should be a test to catch this.

Reply:
I'd like to say that maybe it shouldn't be the responsibility of `SearchResultItem` to check this but it makes sense to have it here. The parent container could make sure that the prop that gets passed to `headingContent` only has one child, but that delves into the app's responsibility.

Reply:
It seems like `<Heading>` should really test for having only one child.  It isn't like this because `Research<em>Now</em>` is counting as two children, but is there harm in requiring something like that to be wrapped in a span?

Reply:
That's true about `Heading` testing it and maybe this part should go there instead of the check happening here. It could be either 1) have every individual component test that it receives one child or 2) check the data at the top level and then pass it down appropriately. If RN needs that heading name then that's fine and enforcing that the child is wrapped in a span is a good idea.

Reply:
The way it's been happening now is that the bottom level component is throwing the error.  However, React isn't tracing its path through (could be the way we set up source maps or something?) so I agree that having a "You passed too many children into SearchPromo" is gonna be a lot more useful than "You passed too many children into Heading".  But I think it's ideal if we could be like, "you passed too many children into Heading, which is called by searchheading, which is called by searchpromo" the way that it would show up if everything was in one project.  
Gonna file another DS card to think about this, but it's yet another "later" thing